src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+db.driver="com.mysql.jdbc.Driver"
```

Comment:
can you remove inverted commas from the values of the properties, its not required

Reply:
I have removed.